backend/controllers/createLinkedDpsUser.js

```diff
@@ -0,0 +1,217 @@
+const { trimObjValues } = require('../utils/utils')
+const {
+  validateUserForSearch,
```

Comment:
can we split this file based on search and create functionality

Reply:
Yes, I can look into that.